test/PublishCommand.js

```diff
@@ -964,49 +983,49 @@ describe("PublishCommand", () => {
         [ChildProcessUtilities, "execSync", {}, [
           { args: ["git tag"] }
         ]],
-       [PromptUtilities, "confirm", { valueCallback: true }, [
```

Comment:
Important as these whitespace changes are, let's fix this separately (I'll approve it and merge immediately)